README.md

```diff
@@ -466,11 +472,11 @@ Tool | About
 
 Reporter | About
 :--- | :----
-[code_climate][code-climate] | Output issues in valid Code Climate JSON (or optionally strictly [Gitlab][gitlab-cc] compatible) to stdout or as a file.
-do_nothing | Does nothing. Useful when piping output to a separate process and no output is desired.
-[json] | Output issues as a JSON list either to stdout or as a file.
-print_to_console | Print the issues to stdout. This is the default reporting plugin if no _profile_ or _level_ are provided.
-[write_jenkins_warnings_ng][jenkins-warnings-ng] | Write Statick results to Jenkins Warnings-NG plugin json-log compatible output.
+[code_climate][code-climate] | Output issues in valid Code Climate JSON (or optionally strictly [Gitlab][gitlab-cc] compatible) to stdout or as a file.<br>Options:<ul><li>files (string): Output issues to a file.</li><li>terminal (string): Output issues to stdout.</li><li>gitlab (string): Output issues in Gitlab Code Climate format.</li></ul>
+do_nothing | Does nothing. Useful when piping output to a separate process and no output is desired. No options.
+[json] | Output issues as a JSON list either to stdout or as a file.<br>Options:<ul><li>files (string): Output issues to a file.</li><li>terminal (string): Output issues to stdout.</li></ul>
```

Comment:
In general I really like all these additions, thank you!

Can you switch these lists from HTML to markdown format?

Reply:
Nevermind, i should've done some googling first. you're good. I found this:
https://stackoverflow.com/questions/19950648/how-to-write-lists-inside-a-markdown-table